rapid7_insightvm/plugin.spec.yaml

```diff
@@ -103,6 +103,11 @@ types:
       title: Type
       type: string
       required: false
+    content:
+      description: Content of the file
+      title: Content
```

Comment:
```suggestion
      title: Contents
```

Reply:
The key returned in the response is `content` should the title not match that?

Reply:
No - contents is the proper form given the context. The key needs to match the input or output, but the title does not have to match.